config/defaults.mk

```diff
@@ -275,6 +276,19 @@ ifeq ($(PETSC_FOUND),YES)
       -L$(abspath $(PETSC_DIR))/lib -lpetsc $(PETSC_LIB)
 endif
 
+SLEPC_ARCH := arch-linux2-c-debug
+SLEPC_DIR := $(MFEM_DIR)/../slepc/$(SLEPC_ARCH)
+SLEPC_VARS := $(SLEPC_DIR)/lib/slepc/conf/slepc_variables
+SLEPC_FOUND := $(if $(wildcard $(SLEPC_VARS)),YES,)
+SLEPC_INC_VAR = SLEPC_CC_INCLUDES
+SLEPC_LIB_VAR = SLEPC_EXTERNAL_LIB_BASIC
+ifeq ($(SLEPC_FOUND),YES)
+    SLEPC_OPT := $(shell set -n "s/$(SLEPC_INC_VAR) = *//p" $(SLEPC_VARS))
+    SLEPC_LIB := $(shell set -n "s/$(SLEPC_INC_LIB) = *//p" $(SLEPC_VARS))
+    PETSC_LIB := -Wl,-rpath,$(abspath $(SLEPC_DIR))/lib\
```

Comment:
Are you sure about it?

Reply:
Probably a copy paste error. I will check this.

Reply:
I fixed it, and tested with make.